tests/pytests/test_fuzzy.py

```diff
@@ -74,7 +74,7 @@ def testFuzzySyntaxError(env):
     for ch in unallowChars:
         error = None
         try:
-            env.cmd('ft.search', 'idx', '%%wor%sd%%' % ch)
+            env.cmd('ft.search', 'idx', f'%wor{ch}d%')
```

Comment:
This is fuzzy!

Reply:
Make sure this is the same

Reply:
The extra % is used to escape the % character itself.